### PR TITLE
- Added tutorial examples in the package com.baeldung.scala.absClassA…

### PR DESCRIPTION
**Pull request by nirmalya-at-baeldung**
Greg Martin, please review.

### CODE REVIEW

File: core-scala/src/test/scala/com/baeldung/scala/absClassAndtraits/AbstractClassAndTraitsTogetherTest.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  "An employee aged over 60 and working for less than 10 years " should {
+    "not be eligible for special allowance " in {
+      employeeFromCalcutta.isEligibleForSpecialAllowance shouldBe false
+    }
+  }
+
+  "A flyer employee from Dublin " should {
+    "have only 2 wings and fly normally " in {
+        flierEmployeeFromDublin.minWings               shouldBe 2
```
Comment (glmartin) [inline core-scala/src/test/scala/com/baeldung/scala/absClassAndtraits/AbstractClassAndTraitsTogetherTest.scala:41]: There should be only 1 space between "flierEmployeeFromDublin.minWings" and "should've"

Comment (nirmalya-at-CW) [reply core-scala/src/test/scala/com/baeldung/scala/absClassAndtraits/AbstractClassAndTraitsTogetherTest.scala]: Point taken, Martin.

Comment (nirmalya-at-baeldung) [reply core-scala/src/test/scala/com/baeldung/scala/absClassAndtraits/AbstractClassAndTraitsTogetherTest.scala]: @glmartin By mistake, the check-in had been from my work account. Apologies. 

I am checking in again, from Baeldung account.

**Comment by nirmalya-at-baeldung** [comment]
@glmartin 

Greg,
Sorry, I didn't understand exactly what you meant. Need help.

**Pull request closed without merging**